doc/builtin_entities.txt

```diff
@@ -0,0 +1,96 @@
+# Builtin Entities
+Minetest registers two entities by default: Falling nodes and dropped items.
+This document describes how they behave and what you can do with them.
+
+## Falling node (`__builtin:falling_node`)
+
+Is created when a node with the special group `falling_node=1` starts to fall.
+Falling nodes can also be created artificially with
+`minetest.spawn_falling_node`.
+
+Spawning a falling node with `/spawnentity` won't work. 
+
+Behaviour:
+
+* Falls down in a straight line (gravity = `movement_gravity` setting)
+* It collides with physical objects and `walkable` nodes
+* If the node group `float=1` is set, it also collides with liquid nodes
+* When it hits a solid (=`walkable`) node, it will try to place itself as a
+  node, replacing the node above
```

Comment:
The following list is not a condition for this statement to apply. They are not related, hence terminating the sentence here clarifies that.
```suggestion
  node, replacing the node above.
```